.github/workflows/trivy.yml

```diff
@@ -1,8 +1,9 @@
 name: dsp-appsec-trivy
 on:
-  pull_request:
+  #pull_request:
```

Comment:
Why is this commented out?

Reply:
it was failing because it wasn't caching it's database properly.